docker-method/Synology.md

```diff
@@ -0,0 +1,74 @@
+# ProximaX Sirius MAINNET Onboarding
+
+## Requirements
+Synology NAS with:
+- 1GB memory or more
+- DSM 6.2+
+
+Ensure that your local network allows inbound/outbound traffic on these ports:
+- 7900/tcp
+- 7901/tcp
+- 7902/tcp
+
+Tested on a Synology DS412+
+
+## Pre-requisite
+Install the following packages using Package Center:
+- Text Editor
+- Docker
+
+Enable SSH access to use the tools:
+- Go to Control Panel
+- Terminal & SNMP
+- Enable SSH Service
+
+## Create folder structure
+- Open File Station
+- Create a folder `proximax-sirius-chain-mainnet` in `docker`
+
+## Download and extract peer information
+- Download: https://github.com/proximax-storage/xpx-mainnet-chain-onboarding/releases/download/release-v0.4.3-buster/public-mainnet-peer-package-v0.4.3.tar.gz
+- Extract `public-mainnet-peer-package-v0.4.3.tar.gz` to a local folder
```

Comment:
using generic version.

`Extract public-mainnet-peer-package-vX.X.X.tar.gz (use latest version) to a local folder`